### ISSUE TITLE
Pittsburgh, PA - Letter to the mayor

### ISSUE THREAD

**Issue by bluemoonkat** [defund12/defund12.org#422]
Mayor's email: bill.peduto@pittsburghpa.gov

Dear Mayor Peduto,

My name is [YOUR NAME] and I am a resident of [NEIGHBORHOOD/BOROUGH/CITY]

From reports of protesters being attacked around the city to the killing of Antwon Rose in East Pittsburgh - the police have shown that they are not doing their jobs to serve and protect all citizens of this city.

In 2020 the Bureau of Police’s budget INCREASED by over $10 million dollars.  The first goal listed on the budget is to improve “boots on the ground” field operations. This is not a budget dedicated to keeping citizens safe and healthy.  

I demand that budgeting funds are moved from the Bureau of Police (~18.8% of Pittsburgh’s Operating Budget) towards programs that support housing, jobs, education, and improve quality of life in our community and for Black Pittsburghers.

Here are some departments from the Operations and Capital budgets that could use this financial support:

Office of Equity - only allocated $1,758,022 (~0.29% of 2020 Operating Budget)
Neighborhood and Community Development - only allocated $7,502,080 (~6.8% of 2020 Capital Budget)
Community Based Organizations - only allocated $500,000 (~0.45% of 2020 Capital Budget)
Neighborhood Employment Centers - only allocated $150,000 (~0.14% of Capital Budget)
Pittsburgh Employment Program - only allocated $150,000 (~0.14% of Capital Budget)
Fair Housing - only allocated $100,000 (~0.09% of Capital Budget)

The lack of community support is evident in the Pittsburgh 2020 budget.  No amount of tweets, photo-ops, bare minimum training, or grand promises will make up for the fact that the city is not monetarily invested in supporting its people. Move to make real change in the city of Pittsburgh by divesting from the Bureau of Police and funneling money back into supporting the community.

Thank you,
[YOUR NAME] [YOUR ADDRESS] [YOUR EMAIL] [YOUR PHONE NUMBER]

**Comment by emilyhuntsman** [comment]
Hi @bluemoonkat I will start by inputting this message into a PR request so it can be uploaded. Would you want to also add the Pittsburgh city council'c emails to the list? If so, I can grab them from the govt website! Please let me know if you have a specific subject line in mind

**Comment by bluemoonkat** [comment]
No preference on subject line!

There's a previous version I wrote that landed a couple days ago that the site has listed as focused on City Council members, you're welcome to add those emails if preferred! Figured I would put in a quick re-write/request for the mayor. I'm okay with whatever is most helpful.

Email list - It's a mix of Council members and their Chiefs of Staff/Assistant Chief for one.
theresa.kail-smith@pittsburghpa.gov,
brosha.tkacheva@pittsburghpa.gov,
district5@pittsburghpa.gov,
district7@pittsburghpa.gov,
erika.strassburger@pittsburghpa.gov,
emilie.yonan@pittsburghpa.gov,
reverend.burgess@pittsburghpa.gov,
daniel.wood@pittsburghpa.gov,
curt.conrad@pittsburghpa.gov,
blake.plavchak@pittsburghpa.gov,
Kim.Salinetro@pittsburghpa.gov,
sally.stadelman@pittsburghpa.gov,

Thanks for doing this!!
